gen/aircraft.py

```diff
@@ -1027,9 +1027,21 @@ def configure_cas(self, group: FlyingGroup, package: Package,
             rtb_winchester=OptRTBOnOutOfAmmo.Values.Unguided,
             restrict_jettison=True)
 
+    def configure_dead(self, group: FlyingGroup, package: Package,
+                       flight: Flight,
+                       dynamic_runways: Dict[str, RunwayData]) -> None:
+        group.task = SEAD.name
+        self._setup_group(group, SEAD, package, flight, dynamic_runways)
+        self.configure_behavior(
+            group,
+            react_on_threat=OptReactOnThreat.Values.EvadeFire,
+            roe=OptROE.Values.OpenFire,
+            rtb_winchester=OptRTBOnOutOfAmmo.Values.ASM,
```

Comment:
DEAD opens up a lot of other options here. This should probably be all A/G weapons aside from the gun, smoke, and illum?

(not a regression so not blocking, just something I thought of when I saw it)